docs/data/charts/zoom-and-pan/ZoomControlled.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from 'react';
+import { LineChartPro } from '@mui/x-charts-pro/LineChartPro';
+import { ZoomData } from '@mui/x-charts-pro/context';
+import { Button } from '@mui/base';
+
+export default function ZoomControlled() {
+  const [zoom, setZoom] = React.useState<ZoomData[]>([
+    {
+      axisId: 'my-x-axis',
```

Comment:
Currently `axisId` is required. Should we allow `{start,end}` to be passed when there is only one zoom enabled?